website/docs/r/lakeformation_resource_lf_tag.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Non-blocking - Might be worth including an `Import` heading just stating that import isn't supported for this resource?